### DISCUSSION TITLE
Facing ECONNREFUSED error using EMQx public broker

### DISCUSSION THREAD

**Discussion by Dhruv-J02** [emqx/emqx#11972] [Q&A] [answered]
I am currently testing few projects based on EMQx public MQTT broker hosted at "broker.emqx.io".
Upon connection I am constantly getting the error : **connect ECONNREFUSED 54.244.173.190:1883**
Kindly guide me through the error if anyone knows this problem as this one is new to me.
Thank you

**Comment by qzhuyan** [comment, marked as answer]
We had some issue last week and it should be solved now.

**Comment by Dhruv-J02** [comment]
No issues. Thank you for your response

Regards,
Dhruv Jaiswal
